### DISCUSSION TITLE
Nominating Joshua Casey for Maintainer role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#142] [Python Team]
I'd like to nominate @joshuatcasey for the Maintainer role for the python projects. He has made notable contributions to multiple projects and has demonstrated thoughtfulness around the future direction of the projects.

@paketo-buildpacks/python-maintainers
@paketo-buildpacks/steering-committee

**Comment by ryanmoran** [comment]
Congrats @joshuatcasey on becoming a Python Maintainer!
